### ISSUE TITLE
Update for EE3+?

### ISSUE THREAD

**Issue by robputer** [croxton/Search_fields#14]
Any chance this could get an update for EE3+ (or most importantly 5)?

**Comment by croxton** [comment]
No, sorry, this project is pretty old now. I would suggest using Low Search instead.
